pkg/kube/inject/testdata/inject/enable-core-dump.yaml.injected

```diff
@@ -204,27 +204,6 @@ spec:
           runAsGroup: 1337
           runAsNonRoot: true
           runAsUser: 1337
-      - args:
```

Comment:
well this seems wrong? what happened?

Reply:
no idea what happened here - i removed the changes and seems to still be passing.

Reply:
ok, i figured it out. sidecar injector was calling a manifest internal function and the format for that had changed.